Does a few things:
1. Adds the sub packages of this project to the workspace members.
   This is required by naersk and probably good practice anyway.
2. Modifies the dependencies to use a local version of json-forensics.
   If we try to use the Git version, it will fail due to not having a
   lock file present in the git repo: "the source ... requires a lock
   file to be present first before it can be used against vendored
   source code".
1. Removes the redis dependecy as it's troublesome and the processing
   feature isn't used in this build.
diff --git a/Cargo.toml b/Cargo.toml
index 9c546b0..0e51ef1 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -11,6 +11,13 @@ build = "build.rs"
 publish = false

 [workspace]
+members = [
+  "common",
+  "general",
+  "general/derive",
+  "server",
+  "server/json-forensics"
+]

 [features]
 default = []
diff --git a/server/Cargo.toml b/server/Cargo.toml
index 2cee289..0535369 100644
--- a/server/Cargo.toml
+++ b/server/Cargo.toml
@@ -13,7 +13,7 @@ license-file = "../LICENSE"
 [features]
 default = ["with_ssl"]
 with_ssl = ["native-tls", "actix-web/tls"]
-processing = ["rdkafka", "redis", "r2d2"]
+processing = ["rdkafka", "r2d2"]

 [dependencies]
 actix = "0.7.9"
@@ -25,7 +25,7 @@ clap = "2.33.0"
 failure = "0.1.5"
 flate2 = "1.0.9"
 futures = "0.1.28"
-json-forensics = { version = "*", git = "https://github.com/getsentry/rust-json-forensics" }
+json-forensics = { path = "./json-forensics" }
 lazy_static = "1.3.0"
 listenfd = "0.3.3"
 log = "0.4.8"
@@ -33,7 +33,6 @@ native-tls = { version = "0.2.3", optional = true }
 parking_lot = "0.9.0"
 r2d2 = { version = "0.8.5", optional = true }
 rdkafka = { version = "0.22.0", optional = true }
-redis = { git = "https://github.com/mitsuhiko/redis-rs", optional = true, branch = "feature/cluster", features = ["cluster", "r2d2"] }
 regex = "1.2.0"
 rmp-serde = "0.13.7"
 sentry = "0.17.0"